kubernetes/istio-install/istio-mixer.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
```

Comment:
Consider a batch job in Kubernetes. It is the source of traffic but has no service associated with it. That's the reason why I only sent `source.uid` (pod IP). Kubernetes does not have a notion of a "borg job" or an "application". Therefore, associating an IP with a service or a job needs to be done outside of the proxy.